templates/minimal/README.md

```diff
@@ -47,19 +47,64 @@ cd minimal-template
 
 ### Build
 
-🔨 Use the following command to build the node without launching it:
+🔨 Use the following command to build the template, which by default
+compiles just the runtime. There is also a `parachain-template-node`
+crate that is able to run and load the runtime but the recommended
+way of starting the template is based on `Omni Node`.
 
 ```sh
 cargo build --release
 ```
 
-🐳 Alternatively, build the docker image:
+### Single-node Development Chain with Omni Node
+
+⬇️  Omni Node can be run by using the `polkadot-omni-node` binary, which can be
+downloaded from [Polkadot SDK releases](https://github.com/paritytech/polkadot-sdk/releases/latest).
+
+* 🔗 Once downloaded, it must be added to the `PATH` environment variable like so:
 
 ```sh
-docker build . -t polkadot-sdk-minimal-template
+export PATH="<path-to-binary>:$PATH"
+```
+
+↩️  The Omni Node needs a runtime chainspec which can be generated based on
+the `minimal-runtime`.
+
+```sh
+# Build the minimal runtime.
+cargo build -p minimal-template-runtime --release
+# Install chain-spec-builder if not installed already.
+cargo install staging-chain-spec-builder
+# Use chain-spec-builder to generate the chain_spec.json file based on the development preset.
+chain-spec-builder create --relay-chain "dev" --para-id 1000 --runtime \
+    <target/release/wbuild/path/to/minimal-template-runtime.wasm> named-preset development
+```
+
+⚙️  The `relay-chain` and `para-id` flags are extra bits of information to configure the node
+for the case of representing a parachain that is connected to a relay chain. They are not relevant
+to minimal template business logic, but they are mandatory information for Omni Node, nonetheless.
+
+🚀 Start Omni Node with manual seal (3 seconds block times) and minimal template runtime based
+chain spec.
+
+```sh
+polkadot-omni-node --chain <path/to/chain_spec.json> --dev-block-time 3000 --tmp
+```
+
+### Single-Node Development Chain with Minimal Template Node
+
+⚙️  Use the following command to build the node as well:
+
+```sh
+cargo build --workspace --release
```

Comment:
A note that here of why ` --workspace` is needed.

Reply:
I re-phrased most of the minimal readme with the hope of clarifying why `--workspace` is needed when used. You can see it here: [ff56fc0](https://github.com/paritytech/polkadot-sdk/pull/5954/commits/ff56fc0e3fd8f4f6680a4e38def8d6d9ca3d2efb).